### PR TITLE
packages: Add swig

### PR DESCRIPTION
**Pull request by TheInfected**

### CODE REVIEW

File: packages/swig/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+TERMUX_PKG_HOMEPAGE=http://swig.org
+TERMUX_PKG_DESCRIPTION="Generate scripting interfaces to C/C++ code"
+TERMUX_PKG_VERSION="3.0.12"
+TERMUX_PKG_BUILD_IN_SRC=yes
+TERMUX_PKG_DEPENDS="boost, pcre, pcre-dev, perl"
```
Comment (xeffyr) [inline packages/swig/build.sh]: 1. Boost is not needed to build swig.
2. Remove pcre-dev - just pcre is enough.

[the remark this answers is not in the document]
Comment (xeffyr) [reply packages/swig/build.sh]: However, I think that perl is not needed here too.

**Comment by yuvrajx** [comment]
@fronwall
What about this?

**Comment by yuvrajx** [comment]
@fornwall
What about this?

**Comment by xeffyr** [comment]
Implemented in [unstable-packages](https://github.com/termux/unstable-packages) repository.
```
pkg install unstable-repo
pkg install swig
```

**Pull request closed without merging**